pyxem/generators/indexation_generator.py

```diff
@@ -32,6 +32,10 @@
 from pyxem.utils.indexation_utils import index_magnitudes
 from pyxem.utils.indexation_utils import match_vectors
 
+from collections import namedtuple
+
+from transforms3d.euler import mat2euler, euler2mat
+import lmfit
```

Comment:
This is a new dependency I think - needs to be added to requirements in: setup.py, appveyor.yml, and .travis.yml, this may well be why the travis and appveyor bots are failing right now.